android/app/src/main/kotlin/coop/polypoly/polypod/AppUpdateReceiver.kt

```diff
@@ -6,8 +6,7 @@ import android.content.Intent
 import androidx.work.OneTimeWorkRequest
 import androidx.work.WorkManager
 import coop.polypoly.polypod.logging.LoggerFactory
-import coop.polypoly.polypod.updatenotification.PushNotificationWorker
-import coop.polypoly.polypod.updatenotification.UpdateNotification
+import coop.polypoly.polypod.core.UpdateNotification
```

Comment:
This new module includes the two it's substituted, right?

Reply:
Nope, `PushNotificationWorker` was just moved one level up, to `coop.polypod.polypod`, for which no explicit `import` is needed here (being in the same package). I don't know why I had grouped those together back when I first implemented it, it doesn't make a lot of sense in hindsight.